core-bundle/src/Csrf/ContaoCsrfTokenManager.php

```diff
@@ -60,4 +62,13 @@ public function canSkipTokenValidation(Request $request, string $tokenCookieName
             && !($request->hasSession() && $request->getSession()->isStarted())
         ;
     }
+
+    public function getFrontendTokenValue(): string
```

Comment:
We could additionally overwrite `getToken(string $tokenName)` with `getToken(string $tokenName = null)` and use the set frontend token name when `$tokenName` wasn't set.

Reply:
I would prefer this 👍

Reply:
I think having a separate method makes this more explicit. Not passing a value basically means _give me what you want_, which is not the same. But it would probably be acceptable for the developer 😂

Reply:
I agree with both. Should we vote?

:+1: = just the `getFrontendTokenValue()` method like it is currently implemented

:tada: = adjust `getToken(string $tokenName = null)` and return the front end token if `null` was given

:rock: = same + `getFrontendTokenValue()` method that basically returns `$this->getToken()->getValue()`

Reply:
Always be explicit. We just had trouble with people doing `Model::findByPk(null)` - let's not repeat the story.

Reply:
So then you intended to vote for :+1: instead of :rocket:?

Reply:
🤦 yeah of course :D